src/main/java/umc/spring/web/controller/StoreController.java

```diff
@@ -0,0 +1,68 @@
+package umc.spring.web.controller;
+
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.Parameters;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import umc.spring.Vaildator.annotation.CheckPage;
+import umc.spring.apiPayload.ApiResponse;
+import umc.spring.repository.ReviewRepository.ReviewRepository;
+import umc.spring.service.ReviewService.ReviewQueryServiceImpl;
+import umc.spring.service.StoreService.StoreQueryServiceImpl;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewRequestDTO;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewResponseDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreRequestDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreResponseDTO;
+
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping("/stores")
+public class StoreController {
+
+    private final StoreQueryServiceImpl storeQueryService;
+
+
+    @PostMapping("{regionId}")
```

Comment:
해당 부분에서 앞에 `/`를 생략하여도 자동으로 경로를 처리하기 때문에 아마 동작은 잘 됬을 거라고 생각되네요. 다만, 코드 일관성을 위해서 `/`를 붙여주는 것이 더 좋아 보입니다!!! 이러면 더 명확하게 URL 경로임을 보여줄 수 있을 거 같네요.
```suggestion
    @PostMapping("/{regionId}")
```